src/concrete/ml/sklearn/base.py

```diff
@@ -1280,8 +1280,41 @@ def __init__(self, n_bits: int):
         #: The model's inference function. Is None if the model is not fitted.
         self._tree_inference: Optional[Callable] = None
 
+        #: Boolean that indicates whether the rounding feature is enabled or not.
+        self._use_rounding: bool = True
+
         BaseEstimator.__init__(self)
 
+    @property
+    def use_optimized_execution(self) -> bool:
```

Comment:
let's remove this @kcelia , we keep just the private property and set that to True/False in tests with pylint disable